client/src/components/DragAndDrop/dNdComponents/itemList.jsx

```diff
@@ -0,0 +1,141 @@
+                                boardActionsInit={boardActionsInit}
+                                boardID={boardID}
+
+                          />
+                        </ScrollContainer>
+                    ) : (
+                            <InnerList columnItems={columnItems} 
+                            dropProvided={dropProvided}
+                                isDraggingOver={dropSnapshot.isDraggingOver}
+                                isDraggingFrom={Boolean(dropSnapshot.draggingFromThisWith)}
+                                classes={classes} boardActions={boardActions}
+                                boardActionsInit={boardActionsInit}
+                                boardID={boardID}
+                             />
+                    )}
+                    {openAddDialog && <Paper className={classes.addCardItem}>
+                        <Grid container wrap="nowrap" spacing={2}>
+                            <Grid item xs={12} sm={12}>
+                                <Typography varaint='h5'> Add title...</Typography>
+                                <TextField
+                                    autoFocus
+                                    id="standard-basic"
+                                    type="text"
+                                    fullWidth
+                                    variant="standard"
+                                    inputRef={inputElement}
+                                    onChange={() => setData(prev => {
+                                        return { ...prev, name: inputElement.current.value }
+                                    })}
+                                />
+                                    <Box style={{display:'flex', flexDirection: 'row', justifyContent: 'space-between', marginTop:'10px'}}>
+                                    <Typography variant='h6' style={{whiteSpace:'nowrap', fontSize:'14px'}}> Select Tag: </Typography>
+                                   
+                                    <Box className={classes.stack}>
+                                        <Avatar onClick={()=>setTag(0)} className={classes.avatarx} style={{background:'#FFF'}} > </Avatar>
+                                        <Avatar onClick={() => setTag(1)} className={classes.avatarx} style={{ background: '#5ACD76' }}> </Avatar>
+                                        <Avatar onClick={() => setTag(2)} className={classes.avatarx} style={{ background: '#FF5D48' }}> </Avatar>
+                                        <Avatar onClick={() => setTag(3)} className={classes.avatarx} style={{ background: '#EDAB1D' }}> </Avatar>
+                                        <Avatar onClick={() => setTag(4)} className={classes.avatarx} style={{ background: '#59B0FF' }}> </Avatar>
+                                        <Avatar onClick={() => setTag(5)} className={classes.avatarx} style={{ background: '#D460F7' }}> </Avatar>
```

Comment:
If we had an array/object that contained the colors, we could `map` over them to simplify this code-block.